shell.nix

```diff
@@ -94,9 +110,11 @@ in
 stdenv.mkDerivation {
   name = "haskell-language-server";
   buildInputs = [
+    git
     gmp
-    zlib
     ncurses
+    openssh
```

Comment:
why do we need openssh?

Reply:
Left over from experiments, we probably don't need it. Let me remove it and see